src/main/java/cart/config/ExternalPropertyConfig.java

```diff
@@ -0,0 +1,16 @@
+package cart.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.annotation.PropertySource;
+
+@Profile(value = "prod")
```

Comment:
"prod"는 상수로 만들면 좋을 것 같습니다. ProfileZone과 같이 따로 정의해두면 좋을 것 같아요. (로컬 환경, 개발 환경, 상용 환경 등등 구분)

Reply:
PROFILE_ZONE이라는 필드명 보다는 아래와 같이 ProfileZone이라는 클래스로 분리하는 것을 생각했었습니다!
```suggestion
    public static final String PROD = "prod";
    public static final String LOCAL = "local";
```
상수 조합이니 private 생성자로 인스턴스 생성을 막아줘도 좋겠네요. 그리고 궁금한 부분이 있는데요. 지금 이 어플리케이션 코드를 받고 로컬로 서버를 띄워볼 수 있어야될 것 같은데 로컬로 실행해보고 싶다고 했을 때는 어떻게 해야하나요?

Reply:
수정했습니다!

음... 로컬로 애플리케이션을 띄울 생각은 못했네요ㅠㅠ db 설정이라든지, 외부 설정이라든지 조금 꼬여 있어서 분리하지 못했습니다.

Reply:
로컬로 애플리케이션이 띄워져야 합니다. 그래야 디버깅할 수 있을테니까요 🙂 보통 로컬 환경, 개발 환경, 상용 환경, 테스트 환경 이런 식으로 구분하는데요. 외부 설정이 많으면 그만큼 신경써줘야하는 부분이 많으니 추후 팀 미션 때 잘 고려해서 작성해보세요!